ua/org.eclipse.ui.cheatsheets/src/org/eclipse/ui/internal/cheatsheets/registry/CheatSheetRegistryReader.java

```diff
@@ -222,12 +222,14 @@ protected CheatSheetElement createCheatSheetElement(IConfigurationElement elemen
 	}
 
 	/**
-	 *	Create and answer a new CheatSheetCollectionElement, configured as a
+	 *	Creates and returns a new CheatSheetCollectionElement, configured as a
 	 *	child of <code>parent</code>
 	 *
 	 *	@return org.eclipse.ui.internal.model.CheatSheetCollectionElement
 	 *	@param parent org.eclipse.ui.internal.model.CheatSheetCollectionElement
-	 *	@param childName java.lang.String
+	 *	@param pluginId java.lang.String
+	 *	@param id java.lang.String
+	 *	@param label java.lang.String
```

Comment:
nit: I see that the documentation was like this before, but maybe it is better to change this into an empty parameter / return value documentation (so that it's obvious that documentation is missing) rather than putting the parameter type into the parameter's documentation?

Reply:
i just want to get the build clean. there are so many unexplained parameters that i doubt anybody will ever document them properly